var/spack/repos/builtin/packages/zookeeper-benchmark/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class ZookeeperBenchmark(Package):
+    """It is designed to measure the per-request latency of a ZooKeeper
+    ensemble for a predetermined length of time"""
+
+    homepage = "http://zookeeper.apache.org"
+    git      = "https://github.com/brownsys/zookeeper-benchmark.git"
+
+    version('master', branch='master')
+
+    variant('zooKeeperVersion', default='3.3.6', description='The client code corresponding to the ZooKeeper version will be found using maven.')
```

Comment:
What if you instead add a dependency on `zookeeper` and use `spec['zookeeper'].version`?

Reply:
You can remove this variant now.